content/hub-services/index.md

```diff
@@ -0,0 +1,125 @@
+
+        However, should your community require a more customized approach, please don't hesitate to [contact us](../join/_index.md) to discuss your specific requirements.
+
+  - block: markdown
+    content:
+      title: Sign-in
+      text: |
+        We support the following authentication and authorization options:
+          - GitHub - with support for GitHub Organization and Teams
+          - CILogon - with support for institutional logins, Google Auth, Microsoft, and ORCID
+          - Shared Password - simple authentication with a global shared password, ideal for workshops and webinar
+        <figure class="videofigure">
+          {{< video src="videos/jupyterhub-admin.mp4">}}
+
+          <figcaption>
+              Powered by <a href="https://jupyterhub.readthedocs.io"><img src="/images/logos/project/jupyterhub.svg" /></a>
+          </figcaption>
+        </figure>      
+
+  - block: features
+    content:
+      title: Compute
+      subtitle: | 
+        <font color="#000000"> We offer a choice of CPU, Memory (from 4GB to 512GB) and GPU configurations for all types of workloads, with added support for Dask Gateway for task parallelization. 
+
+        We can also co-locate your hub's compute next to your cloud data to improve performance and costs, with AWS and Google Cloud Platform supported out-of-the box, and Azure configurations available on request.</font>
+
+      items:
+        - icon: google-cloud
+          icon_pack: custom
+          name: Google Cloud
+          description: ""
+        - icon: azure 
+          icon_pack: custom
+          name: Microsoft Azure
+          description: ""
+        - icon: aws
+          icon_pack: custom 
+          name: Amazon Web Services
+          description: ""
+
+  - block: markdown
+    content:
+      title: Software Stack
+      subtitle: ""
+      text: |
+        2i2c supports a number of standard community-maintained images for research and education including Pangeo Notebook, SciPy, Julia, and Rocker with RStudio.
+
+        Additionally, hub users can configure their hubs with any existing pre-built image hosted on [Docker Hub](https://hub.docker.com/) or [quay.io](https://quay.io).
+
+        Hub administrators can also provide their own self-maintained images, should they wish to do so.
+        
+        {{< softwarestacklogos >}}
+
+  - block: markdown
+    content:
+      title: Storage
+      subtitle: ""
+      text: |
+        Hubs can be configured with a choice of persistent home directories and different levels of read-write permissions (admins only, or everyone).
+
+        Additionally, object storage can be configured for access to scratch, persistent, or pre-existing buckets, as needed.
+
+  - block: markdown
+    content:
+      title: Sharing
+      subtitle: ""
+      text: |
+        Learning and discovery through interactive cloud computing is more powerful when shared.
```

Comment:
Can we please change the "is" to "are"? My fault, it was my mistake on the text I provided

Reply:
```suggestion
        Learning and discovery through interactive cloud computing are more powerful when shared.
```